services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -109,11 +109,16 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         # NOTE: suffix '_LOGLEVEL' is used overall
     )
 
-    WEBSERVER_LOG_FORMAT_LOCAL_DEV_ENABLED: bool = Field(
+    WEBSERVER_LOG_FORMAT_LOCAL_DEV_ENABLED: dict = Field(
         default=False,
         env=["WEBSERVER_LOG_FORMAT_LOCAL_DEV_ENABLED", "LOG_FORMAT_LOCAL_DEV_ENABLED"],
         description="Enables local development log format. WARNING: make sure it is disabled if you want to have structured logs!",
     )
+    WEBSERVER_LOG_FILTER_MAPPING: dict = Field(
+        default={},
+        env=["WEBSERVER_LOG_FILTER_MAPPING", "LOG_FILTER_MAPPING"],
```

Comment:
NOTE that the `WEBSERVER_LOG_FILTER_MAPPING` (or any other `$(SERVICENAME)_LOG_FILTER_MAPPING`) environment variable is not usable because it has not been defined or exposed in the `docker-compose.yml` file for the relevant services.

Using alias mappings for these variables is outdated and constrained by our current `docker-compose` setup:

1. **Both Variables Exposed:**
   ```yaml
   environment:
     WEBSERVER_LOG_FILTER_MAPPING=${WEBSERVER_LOG_FILTER_MAPPING}
     LOG_FILTER_MAPPING=${LOG_FILTER_MAPPING}
   ```
   In this configuration, `LOG_FILTER_MAPPING` will be ignored in favor of `WEBSERVER_LOG_FILTER_MAPPING`, regardless of its value.

2. **Mapped Differently in `.env` for Each Service:**
   ```yaml
   environment:
     WEBSERVER_LOG_FILTER_MAPPING=${LOG_FILTER_MAPPING}
   ```
   Here, `LOG_FILTER_MAPPING` is mapped directly to `WEBSERVER_LOG_FILTER_MAPPING` in the `.env` file, allowing customization for each service.

3. **Direct One-to-One Mapping:**
   ```yaml
   environment:
     WEBSERVER_LOG_FILTER_MAPPING=${WEBSERVER_LOG_FILTER_MAPPING}
   ```
   This setup maintains a straightforward one-to-one mapping between `WEBSERVER_LOG_FILTER_MAPPING` and its environment variable, without any aliasing.